db/migrate/20210909105614_add_last_login_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastLoginAtToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :last_login_at, :timestamp
```

Comment:
- Is is okay to record this about our users? Do we need to update privacy policy or anything else.
- Might be useful addition to for admins?
- Could we switch to some other column?

Reply:
Discussed on standup and we felt there wasn't any issues with storing this - @garethrees am I recalling our discussion on Friday correct?